### ISSUE TITLE
Create a .snap package of loqui

### ISSUE THREAD

**Issue by pharaone** [loqui/im#1205]
Canonical is now hardworking for making Ubuntu touch a snap based system, loqui should be a snap too?

**Comment by nfsprodriver** [comment]
Yes, good point! This will be a topic, when UT has migrated to snap system.

**Comment by Honk2** [comment]
Is it possible to make the snap package a buil target?
